rotkehlchen/rotkehlchen.py

```diff
@@ -170,6 +170,24 @@ def __init__(self, args: argparse.Namespace) -> None:
         self.shutdown_event = gevent.event.Event()
         self.migration_manager = DataMigrationManager(self)
 
+    def kill_running_tx_query_tasks(self) -> None:
+        """Checks for running greenlets related to transactions query and kills them."""
+        assert self.task_manager is not None
+        log.debug(
+            f'Attempting to kill running transaction queries tasks '
+            f'from api task greenlets {self.api_task_greenlets} and '
+            f'task manager running greenlets {self.task_manager.running_greenlets}',
+        )
+        for greenlet in self.api_task_greenlets:
+            if greenlet.args[0].__func__.__name__ == '_get_evm_transactions':
```

Comment:
no no. Yesterday I told you. Check the full name including the class. This way you will trigger for any class's `_get_evm_transactions`